### PR TITLE
docs(README): update grammar / wording

### PR DESCRIPTION
**Pull request by Maritsu**
changed you're to your

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,5 +19,5 @@
 
 ## Usage
 
-1. Copy the contents of you're prefered style (ex. `mocha.conf`) into `$HOME/.config/conky/<stylename>.conf`.
+1. Copy the contents of your prefered style (ex. `mocha.conf`) into `$HOME/.config/conky/<stylename>.conf`.
```
Comment (sgoudham) [inline README.md]:
```suggestion
1. Copy the contents of your preferred style (ex. `mocha.conf`) into `$HOME/.config/conky/<flavour>.conf`.
```

File: README.md [hunk 2]
```diff
@@ -19,5 +19,5 @@
 
 ## Usage
 
-1. Copy the contents of you're prefered style (ex. `mocha.conf`) into `$HOME/.config/conky/<stylename>.conf`.
+1. Copy the contents of your preferred style (ex. `mocha.conf`) into `$HOME/.config/conky/<flavour>.conf`.
 2. Run using `conky -c $HOME/.config/conky/<stylename>.conf`
```
Comment (sgoudham) [inline README.md]:
```suggestion
2. Run using `conky -c $HOME/.config/conky/<flavour>.conf`
```

**Review by sgoudham — APPROVED** [review]
Nice catch, thanks for the PR!

File: README.md [hunk 3]
```diff
@@ -19,5 +19,5 @@
 
 ## Usage
 
-1. Copy the contents of you're prefered style (ex. `mocha.conf`) into `$HOME/.config/conky/<stylename>.conf`.
-2. Run using `conky -c $HOME/.config/conky/<stylename>.conf`
+1. Copy the contents of your preferred style (ex. `mocha.conf`) into `$HOME/.config/conky/<flavour>.conf`.
+2. Run using `conky -c $HOME/.config/conky/<flavour>.conf`
```
Comment (uncenter) [inline README.md:23]:
```suggestion
1. Copy the contents of your preferred style (ex. `mocha.conf`) into `$HOME/.config/conky/<flavor>.conf`.
2. Run using `conky -c $HOME/.config/conky/<flavor>.conf`
```

Comment (Maritsu) [reply README.md:23]: This is a matter of British vs American spelling and is (i think) outside of the scope of this PR, as both are correct.

Comment (uncenter) [reply README.md:23]: We typically use the American spelling for consistency across Catppuccin repos, but yeah it was just a suggestion not a requirement.

**Pull request merged**